tests/queries/0_stateless/02483_add_engine_full_column_to_system_databases.reference

```diff
@@ -0,0 +1 @@
+Replicated(\'some/path/replicated_database_test\', \'shard_1\', \'replica_1\')
```

Comment:
```suggestion
Replicated(\'some/path/default/replicated_database_test\', \'shard_1\', \'replica_1\')
```

Reply:
(generated database name will be replaced with `default` in the test output before comparing it with the reference file)